Description: Do not attempt to compile removed mbedtls source
 mbedtls is not currently used, and including an embedded copy of its source
 could cause false-positives when scanning for security issues. This code is
 stripped from our orig.tar (at the request of the Ubuntu security team), so
 we also need to disable the build.
Author: dann frazier <dannf@debian.org>
Forwarded: not-needed
Last-Update: 2024-02-25

--- a/CryptoPkg/CryptoPkg.dec
+++ b/CryptoPkg/CryptoPkg.dec
@@ -27,10 +27,6 @@
   Library/OpensslLib/openssl/providers/implementations/include
   Library/OpensslLib/OpensslGen/include
   Library/OpensslLib/OpensslGen/providers/common/include
-  Library/MbedTlsLib/Include
-  Library/MbedTlsLib/mbedtls/include
-  Library/MbedTlsLib/mbedtls/include/mbedtls
-  Library/MbedTlsLib/mbedtls/library
 
 [LibraryClasses]
   ##  @libraryclass  Provides basic library functions for cryptographic primitives.
@@ -49,7 +45,6 @@
   ##  @libraryclass  Provides library functions from the openssl project.
   #
   OpensslLib|Private/Library/OpensslLib.h
-  MbedTlsLib|Private/Library/MbedTlsLib.h
 
   ##  @libraryclass  Provides compiler intrinsic functions required to link openssl project.
   #
